qa/workunits/cephtool/test.sh

```diff
@@ -574,6 +574,22 @@ function test_auth_profiles()
   rm -f client.xx.keyring client.xx.keyring.2
 }
 
+function test_mon_caps()
+{
+  ceph-authtool --create-keyring /etc/ceph/ceph.client.bug.keyring
+  chmod +r /etc/ceph/ceph.client.bug.keyring
+  ceph-authtool /etc/ceph/ceph.client.bug.keyring -n client.radosgw.bug --gen-key
+  ceph -k /etc/ceph/ceph.client.admin.keyring auth add client.radosgw.bug -i /etc/ceph/ceph.client.bug.keyring
+  
+  if ! cat /etc/ceph/ceph.conf | grep client.radosgw.bug; then
+    printf "[client.radosgw.bug]\n \
+        keyring = /etc/ceph/ceph.client.bug.keyring \n" >> /etc/ceph/ceph.conf  
+  fi
+  
```

Comment:
this test suite runs without rgw installed and the fix does not relate to rgw. Could you make it so the test does not require rgwadmin ?